src/hooks/useReceivedNutzaps.ts

```
@@ -0,0 +1,228 @@
+
+          // Verify the tokens are P2PK locked to the user's pubkey
+          // (This is a basic check - in a production app you'd do a more thorough validation)
+          for (const proof of proofs) {
+            // Check if the token is P2PK locked properly
+            try {
+              const secret = JSON.parse(proof.secret);
+              if (!(Array.isArray(secret) &&
+                secret[0] === 'P2PK' &&
+                secret[1]?.data?.startsWith('02'))) {
+                // Not properly P2PK locked
+                continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)